README.md

```diff
@@ -25,16 +25,33 @@ bundle exec jekyll serve
 
 We are using the theme [Just the docs](https://github.com/pmarsceill/just-the-docs).
 
-Yo have to create a markdown file and follow the next structure:
+You have to create markdown files inside the folder struction, following the structure:
 
 ```
 ---
 layout: default
 title: A title
 parent: Name of the parent
 grand_parent: Name of the grand parent if needed
-has_child: true
+has_child: (bool)  
+has_toc: (bool) 
 ---
 
 Excepteur consequat reprehenderit velit mollit occaecat velit esse laborum eiusmod irure eu. Ex fugiat officia elit qui veniam nostrud nisi est. Est reprehenderit nisi sunt non nulla irure aute sunt do.
-```
\ No newline at end of file
+```
+
+Where: 
```

Comment:
Do we really need this? cant that be better to link to the documentation of the https://github.com/pmarsceill/just-the-docs? People using this repo should at least be able to read documentation as they are expected to know a bit about how the code work.

Reply:
It's already there and it's a small peice of at-a-glance info. Not sure we lose anything by having 2 bullets of key info (rather than reading the whole documentation)